lib/streamlit/server/Server.py

```diff
@@ -228,7 +228,7 @@ def _create_app(self):
         tornado.web.Application
 
         """
-        base = config.get_option("server.baseUrlPath")
+        base = config.get_option("server.baseUrlPath").strip("/")
```

Comment:
Nice catch!

Can you move this into `make_url_path_regex`?

Reply:
I've replaced this with stripping `/` from all path components in `make_url_path_regex`.